internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -399,23 +399,33 @@ func (srv *Server) GetAllChangelogEntries(w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	if (len(offset_param) == 0 &&  len(limit_param) == 0){
+		limit := math.MaxInt32
```

Comment:
We should have to different backend functions. One that supports an offset and limit, `GetChangeLogEntries`, and one that returns all entries, `GetAllChangeLogEntries`. We don't want to arbitrarily set the limit to what we think is the max. Instead we should query for what we want exactly.